app/controllers/application_controller.rb

```diff
@@ -5,6 +5,22 @@ class ApplicationController < Sinatra::Base
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "password_security"
   end
 
+  get '/' do 
+    erb :index
+  end
+
+  helpers do
+    def logged_in?
+      !!session[:id]
```

Comment:
Hi @aviflombaum Does this mean we also need to change the logout method to include @current_user = nil? Otherwise once that variable is set there is nothing to clear it? @lisamarie616